app/views/hq/partners/_form.html.erb

```diff
@@ -2,32 +2,35 @@
   <%= render 'shared/errors', object: f.object %>
   <div class="form-group">
     <%= f.label :name %>
-    <%= f.text_field :name, class: "form-control" %>
+    <%= f.text_field :name, class: 'form-control' %>
```

Comment:
if we have agreed to allow people to submit code in their preferred style can we not change previously submitted code.